tests/focusedMethods.js

```diff
@@ -0,0 +1,7 @@
+let focusedMethods = ['web'];
```

Comment:
I'm quite concerned with this, for a few reasons:

1) This PR creates a bunch of did document files that are not used. I understand we have to start somewhere and cannot do everything in one PR, but I'd much rather:
    - have only a test did document for did:web in `test-dids` since it's the method that is currently being tests in CI
    - **document the steps for how someone can add another did method to the test suites**: at least
        1) reference this file in the README
        2) explain how one can download a test did document (looks like this is already done)
        3) explain where one needs to add any did method specific context for the tests to run (eg elem needs https://docs.element-did.com/contexts/sidetree/sidetree-v0.1.jsonld) 

2) The body of the PR says all tests are focused on `did:key` but it looks like they are focused on `did:web` instead.

Reply:
I updated the PR description and removed all the untested did method... I agree with all your comments.

> 3\. explain where one needs to add any did method specific context for the tests to run (eg elem needs https://docs.element-did.com/contexts/sidetree/sidetree-v0.1.jsonld)

We won't be adding any did methods to this repo that use contexts that are hosted like this... thats part of what we are trying to fix here.

Reply:
> 2. The description of the PR says all tests are focused on did:key but it looks like they are focused on did:web instead.

This is still true. We have a did key did document but are not running the tests on it

Reply:
its used to run the tests, because we need it to verify the signature on did:web.

Reply:
Ok, but the PR still says "focused on did:key for now" whereas we are running the tests on "did:web"

Reply:
This is not longer true... we have did:key for testing and did:web for testing... all other did methods have been removed for now. can I resolve this?